rpc/rpc-wrappers.go

```diff
@@ -1016,6 +1020,30 @@ func NewStorageAuthorityServer(rpc RPCServer, impl core.StorageAuthority) error
 		return
 	})
 
+	rpc.Handle(MethodCountCertificatesRange, func(req []byte) (response []byte, err error) {
+		var countReq struct {
```

Comment:
`countReq` should be defined once in this file and shared by both ends of the RPC code.